sunpy/instr/goes.py

```diff
@@ -88,31 +145,35 @@ def get_goes_event_list(timerange, goes_class_filter=None):
 
     return goes_event_list
 
-def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH):
+def calculate_temperature_em(goeslc, abundances="coronal",
+                             download=False, download_dir=None):
```

Comment:
We need to use None as the default arguments so that the constant is not compiled into the function definition, which is bad from a technical point of view and also is really confusing in the sphinx docs.

If you are not aware of what I mean try this:

```python
def test(mylist=[]):
    mylist.append(1)
    print(mylist)
```

try calling that function a few times.